dspace/config/modules/oai.cfg

```diff
@@ -42,7 +42,20 @@ oai.bitstream.baseUrl = ${dspace.ui.url}
 oai.config.dir = ${dspace.dir}/config/crosswalks/oai
 
 # Description
-oai.description.file = ${dspace.dir}/config/crosswalks/oai/description.xml
+oai.description.file.0 = ${dspace.dir}/config/crosswalks/oai/description.xml
+oai.description.file.1 = ${dspace.dir}/config/crosswalks/oai/description-olac.xml
+
+# Values below used in description-olac.xml for identifying repository in OAI-PMH
+description.archiveURL = http://lindat.cz/
+description.participant.name = Name Surname
+help.mail = dspace-help@ufal.mff.cuni.cz
+description.institution = Institute of Formal and Applied Linguistics
+description.institutionURL = http://ufal.mff.cuni.cz
+description.shortLocation = Prague, Czech Republic
+description.location = Malostranské náměstí 25,118 00 Prague, Czech Republic
+description.synopsis = http://lindat.cz
+description.access = https://lindat.mff.cuni.cz/repository/xmlui/page/about
```

Comment:
Use https://lindat.mff.cuni.cz/repository/static/about - because we changed path to static files

Reply:
done